npe2/io_utils.py

```diff
@@ -38,9 +38,10 @@ def read(path: PathLike, *, plugin_name: Optional[str] = None) -> List[LayerData
 
 
 def read_get_reader(
-    path: PathLike, *, plugin_name: Optional[str] = None
+    path: Union[str, Sequence[str]], *, plugin_name: Optional[str] = None
 ) -> Tuple[List[LayerData], ReaderContribution]:
     """Variant of `read` that also returns the `ReaderContribution` used."""
+    _ensure_str_or_seq_str(path)
```

Comment:
this (and `read`) is actually one place where I think we should allow Path objects.  This is part of the public api that someone (besides napari) can use to take advantage of napari plugins without having to open up napari.  In other words, these functions are there to satisfy the requests in https://github.com/napari/napari-plugin-engine/issues/24

Reply:
I'm happy to cast here, though from a [github search](https://github.com/search?q=read_get_reader&type=code) it's only used on a test Repo of Juan, so I _think_ we still have the opportunity to tell users that they should pass in strings.